simplivity/resources/policies.py

```diff
@@ -73,6 +73,23 @@ def get_by_data(self, data):
         """
         return Policy(self._connection, self._client, data)
 
+    def create(self, name, flags=None, timeout=-1):
+        """ create the new policy
+
+        Args:
+            flags: Dictionary of filters, example: {'cluster_group_id': 'cluster_group_id'}
```

Comment:
change docstring

filters: Dictionary of flags. Example: {'cluster_group_id': 'cluster_group_id'}

Reply:
Updated string as per suggestion